akka-sample-cluster-java/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>akka-sample-cluster-java</artifactId>
+  <groupId>com.typesafe.akka.samples</groupId>
+  <name>Akka Cluster in Java</name>
+  <version>empty</version>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <profiles>
+    <profile>
+      <id>2.12.4</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <properties>
+        <scalaBinaryVersion>2.12</scalaBinaryVersion>
+      </properties>
+    </profile>
+    <profile>
+      <id>2.11.8</id>
```

Comment:
I don't see a reason for providing two alternatives here. Java dev shouldn't care, and we can make the build simpler by only using 2.12

Reply:
I agree, we should remove it from the existing poms as well, I guess this was just copy-pasta from there?

Reply:
shall we add gradle also, while we are at it?

Reply:
That would be nice, but maybe make that a separate issue to not overwhelm @raboof with somewhat unrelated work in this PR